Dockerfile.amd64

```diff
@@ -70,6 +80,8 @@ MAINTAINER Christian Berger "christian.berger@gu.se"
 
 #Start microservice
 WORKDIR /usr/bin
+# This will make the image heavy... Do we need all of them?
+# This could be slimmed down
 COPY --from=builder /usr/local/include/ /usr/include/
 COPY --from=builder /usr/local/lib/ /usr/lib/
```

Comment:
For the libs: you can run `ldd` to see what the dependent shared libraries are to carry over.

Reply:
Agreed, I'll leave this as homework for the cfsd team.